includes/html/pages/routing/bgp.inc.php

```diff
@@ -249,7 +249,7 @@
             $peer_type = "<span style='color: #00f;'>iBGP</span>";
         } else {
             $peer_type = "<span style='color: #0a0;'>eBGP</span>";
-            if ($peer['bgpPeerRemoteAS'] >= '64512' && $peer['bgpPeerRemoteAS'] <= '65535') {
+            if (($peer['bgpPeerRemoteAs'] >= '64512' && $peer['bgpPeerRemoteAs'] <= '65534') || ($peer['bgpPeerRemoteAs'] >= '4200000000' && $peer['bgpPeerRemoteAs'] <= '4294967294')) {
```

Comment:
Are you sure the lower number isn't wrong here too?
In another place it is > 64512 and < 65535